tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -184,6 +184,7 @@ namespace Datadog.Trace
     public static class HttpHeaderNames
     {
         public const string DatadogTags = "x-datadog-tags";
+        public const string InvocationError = "x-datadog-invocation-error";
```

Comment:
I don't think this should be public - we don't expect users to use it

Reply:
Good catch! Same for `SpanId` added previously in #2233, not released yet.

Reply:
✅